patches/react-scripts+5.0.1.patch

```diff
@@ -0,0 +1,138 @@
+diff --git a/node_modules/react-scripts/config/webpackDevServer.config.js b/node_modules/react-scripts/config/webpackDevServer.config.js
+index 522a81b..312f2b4 100644
+--- a/node_modules/react-scripts/config/webpackDevServer.config.js
++++ b/node_modules/react-scripts/config/webpackDevServer.config.js
+@@ -1,11 +1,3 @@
+-// @remove-on-eject-begin
+-/**
+- * Copyright (c) 2015-present, Facebook, Inc.
```

Comment:
Why have you removed the comments? Others will need to reference this in future and without the context it will be harder.